src/test-provider/util.ts

```diff
@@ -0,0 +1,11 @@
+import { Range } from "vscode-languageclient/node";
+import * as vscode from "vscode";
+
+export function toVscodeRange(range: Range): vscode.Range {
```

Comment:
I don't know :(
One of them is just an interface (you can think about it as a typed JSON without any runtime overhead) and the second one is a class (classes have runtime overhead). VS code testing API expects the second.

It could be dangerous if you mix those two. For example, some internals might expect class and try to invoke a function on it. However, because you used wrong type in you interface Typescript didn't produce compiler error and bum, you got runtime error :(